### DISCUSSION TITLE
v3.2.111

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#14202] [General]
## What's Changed
* Bump webpack from 5.91.0 to 5.94.0 by @dependabot in https://github.com/filamentphp/filament/pull/14106
* Bump micromatch from 4.0.7 to 4.0.8 in /docs-assets/app by @dependabot in https://github.com/filamentphp/filament/pull/14107
* Bump elliptic from 6.5.5 to 6.5.7 by @dependabot in https://github.com/filamentphp/filament/pull/14108
* Use assertEqualsCanonicalized() for summary testing by @cheesegrits in https://github.com/filamentphp/filament/pull/14124
* Adding Page Sub Navigation Hooks by @dbpolito in https://github.com/filamentphp/filament/pull/14101
* Update cancel button label to uppercase (Dutch language) by @CodeWithDennis in https://github.com/filamentphp/filament/pull/14156
* The toggleAllCheckboxes function should not check a disable checkbox by @hamidroohani in https://github.com/filamentphp/filament/pull/14178
* fix: Unmounting actions dispatches close-modal when modal is already closed by @wout-o in https://github.com/filamentphp/filament/pull/14200

## New Contributors
* @dbpolito made their first contribution in https://github.com/filamentphp/filament/pull/14101
* @hamidroohani made their first contribution in https://github.com/filamentphp/filament/pull/14178
* @wout-o made their first contribution in https://github.com/filamentphp/filament/pull/14200

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.110...v3.2.111

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.111'>v3.2.111</a>.</em>

**Comment by iHabibullahBahar** [comment]
<img width="1440" alt="Screenshot 2024-09-12 at 1 51 48 AM" src="https://github.com/user-attachments/assets/c01c32fa-4bdf-4778-8c69-fe80df11f0f7">

When I am trying to login it is showing the issue.

**Reply by leandrocfe** [reply]
You mean, in that version? Does it work in 3.2.110?
